go/cmd/vtctldclient/command/vreplication/movetables/movetables.go

```diff
@@ -49,6 +49,7 @@ func registerCommands(root *cobra.Command) {
 	create.Flags().BoolVar(&createOptions.AtomicCopy, "atomic-copy", false, "(EXPERIMENTAL) A single copy phase is run for all tables from the source. Use this, for example, if your source keyspace has tables which use foreign key constraints.")
 	create.Flags().StringVar(&createOptions.WorkflowOptions.TenantId, "tenant-id", "", "(EXPERIMENTAL) The tenant ID to use for the MoveTables workflow into a multi-tenant keyspace.")
 	create.Flags().BoolVar(&createOptions.WorkflowOptions.StripShardedAutoIncrement, "remove-sharded-auto-increment", true, "If moving the table(s) to a sharded keyspace, remove any auto_increment clauses when copying the schema to the target as sharded keyspaces should rely on either user/application generated values or Vitess sequences to ensure uniqueness.")
+	create.Flags().StringSliceVar(&createOptions.WorkflowOptions.Shards, "shards", nil, "(Multi-tenant migrations only) Specify that vreplication streams should only be created on this subset of target shards. Warning: you should first ensure that all rows on the source route to the specified subset of target shards using your VIndex of choice or you could lose data during the migration.")
```

Comment:
```suggestion
	create.Flags().StringSliceVar(&createOptions.WorkflowOptions.Shards, "shards", nil, "(EXPERIMENTAL: Multi-tenant migrations only) Specify that vreplication streams should only be created on this subset of target shards. Warning: you should first ensure that all rows on the source route to the specified subset of target shards using your Vindex of choice or you could lose data during the migration.")
```

Reply:
It feels like this should be called `target-shards` given that we already have a `source-shards` option. I wonder what happens if someone specifies both of these options. Do we want to prevent that? Or ignore `source-shards` if `tenant-id` is specified?

Reply:
Also, may as well expand `EXPERIMENTAL` in the flag help for tenant-id (line 50) to `EXPERIMENTAL: Multi-tenant migrations only`.

Reply:
That's unfortunate. I guess `--shards` made sense for shard-by-shard because it's very clear which shard you are currently migrating. That is not the case for multi-tenant. What do you think about adding `--target-shards` for multi-tenant migrations, and "migrating" shard-by-shard migrations to use that in the next release (deprecate old flag, add new flag, respect both, delete in next release etc.)? Or we can even wait to do that until multi-tenant is no longer experimental. I'll approve anyhow, it's not worth holding up this PR, but I feel the UX will be better and clearer if we call it `--target-shards`.

Reply:
I agree `--target-shards` makes more sense here, and we may as well do it now rather than go through a deprecate cycle.  

I will update the code for this and we can decide on modifying the other usages at a later date.